src/bundle/Resources/config/default_parameters.yaml

```diff
@@ -1,5 +1,4 @@
 parameters:
-    ezsettings.admin_group.content_edit.templates.create_draft: '@@ezdesign/content/create.html.twig'
     ezsettings.admin_group.user_edit.templates.update: '@@ezdesign/user/edit.html.twig'
```

Comment:
Maybe it is worth to move those lines to `src/bundle/Resources/config/ezplatform_default_settings.yaml`